src/VirtoCommerce.CartModule.Data/Repositories/CartDbContext.cs

```diff
@@ -189,6 +189,13 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
 
             #endregion ConfigurationItemFile
 
+            #region CartSharingSetting
+            modelBuilder.Entity<CartSharingSettingEntity>().ToTable("CartSharingSetting").HasKey(x => x.Id);
+            modelBuilder.Entity<CartSharingSettingEntity>().Property(x => x.Id).HasMaxLength(IdLength).ValueGeneratedOnAdd();
```

Comment:
These two line can be replaced with `modelBuilder.Entity<CartSharingSettingEntity>().ToAuditableEntityTable("CartSharingSetting")`